### PR TITLE
Modifies Sentry logging for SSO

### PR DESCRIPTION
**Pull request by asg5704**
## Description
Discovery during a Sentry Hackathon for the Identity team.  There is an issue with Sentry appropriately tagging/ignoring exceptions as it relates to the Identity team.  This PR updates Sentry logging in the `keepAliveSSO.js` file and ensures `monitoring/sentry.js` attaches stack traces appropriately.

## Testing done
manual

## Acceptance criteria
- [x] Sentry filters out unneeded log events that are of type `info` as determined by the team. (ie TypeError: The network connection was lost., etc)
- [x] Sentry continues to log `error` and `warning` levels.
- [x] Sentry attaches the stack traces if they are available

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/utilities/sso/keepAliveSSO.js [hunk 1]
```diff
@@ -2,25 +2,53 @@ import * as Sentry from '@sentry/browser';
...
+      LEVEL: Sentry.Severity.Error,
     },
   };
+
+  const LEVEL = caughtExceptions[message]
+    ? caughtExceptions[message].TYPE
+    : caughtExceptions.default.TYPE;
+
+  // Doesn't log "info" to Sentry
+  if (LEVEL !== Sentry.Severity.Info) {
```
Comment (bosawt) [inline src/platform/utilities/sso/keepAliveSSO.js:34]: Would it make sense to either make this an env var or at least put at the top of the file, something like `SENTRY_LOG_THRESHOLD = Sentry.Severity.Info`, or `SENTRY_LEVELS_TO_LOG = [Sentry.Severity.Warn Sentry.Severity.Error]`, just wondering if we'd like a way to customize this in the future

**Review by bosawt — APPROVED** [review]
Looks good to me! Definitely a lot cleaner. I had one potential suggestion but I consider it non-blocking.

Also, I didn't to any direct tests with this, since I wouldn't be testing locally, hopefully we'll be able to run a couple tests on staging and confirm!

File: src/platform/utilities/sso/keepAliveSSO.js (hunk 1 above)
Comment (AGarcia-Clarity) [reply src/platform/utilities/sso/keepAliveSSO.js:34]: Yeah I think this is a good point for extensibility in the future.

**Review by kfrz — APPROVED** [review]
LGTM!

**Review by billfienberg — APPROVED** [review]
LGTM

**Pull request merged**